packages/react-api/src/Api.tsx

```diff
@@ -243,7 +243,7 @@ async function createApi (apiUrl: string, signer: ApiSigner, isLocalFork: boolea
       await setStorage(chopsticksFork, {
         System: {
           Account: [
-            [['5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY'], { data: { free: 1000 * 1e12 }, providers: 1 }]
+            [['5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY'], { data: { free: 100000 * 1e12 }, providers: 1 }]
```

Comment:
This will not work because it will exceed the the allowed max integer: `Number needs to be an integer <= Number.MAX_SAFE_INTEGER, i.e. 2 ^ 53 - 1`. In this case best if you change the value to a string, i.e - '100000000000000000'

Reply:
Ah, okay! I guess 5000 is still within the bounds and should be good for almost all key things to test on Kusama. I guess this suffices for now.